### PR TITLE
boards: arm: Add board definition for Adafruit Feather nRF52840 Express

### PR DESCRIPTION
**Pull request by tsvehagen**
Add support for adafruit_feather_nrf52840 board.

Signed-off-by: Tobias Svehagen <tobias.svehagen@gmail.com>

### CODE REVIEW

**Review by carlescufi — APPROVED** [review]

**Comment by MaureenHelm** [comment]
Please fix the shippable failures

File: boards/arm/adafruit_feather_nrf52840/Kconfig.board [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Adafruit Feather nRF52840 Express board configuration
+
+# Copyright (c) 2020 Tobias Svehagen
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_ADAFRUIT_FEATHER_NRF52840
+	bool "Feather nRF52840 Express Board"
```
Comment (anangl) [inline boards/arm/adafruit_feather_nrf52840/Kconfig.board]: Using "Adafruit Feather nRF52840 Express" as this prompt would make it more consistent with what is used for most of other boards.  The adafruit_feather_stm32f405 board seems to be rather an exception to the rule in this regard that should be corrected perhaps.

**Review by anangl — APPROVED** [review]

File: boards/arm/adafruit_feather_nrf52840/Kconfig.board (hunk 1 above)
Comment (tsvehagen) [reply boards/arm/adafruit_feather_nrf52840/Kconfig.board]: Done

**Review by ioannisg — APPROVED** [review]

**Comment by ioannisg** [comment]
@tsvehagen is it possible to fix the two check-patch warnings in test_adc.c?

**Comment by tsvehagen** [comment]
> @tsvehagen is it possible to fix the two check-patch warnings in test_adc.c?

Yea I changed the indentation to tabs

**Pull request merged**